resources/js/components/EmailVerify.jsx

```diff
@@ -0,0 +1,42 @@
+// VerifyEmail.js
+
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+const VerifyEmail = () => {
+    const { token } = useParams();
+    const [loading, setLoading] = useState(true);
+    const [message, setMessage] = useState('');
+
```

Comment:
thanks for the comments is good practice

Reply:
Done -> Added more comments.